android/src/main/java/com/reactlibrary/RNSpotifyRemoteAppModule.java

```diff
@@ -40,6 +40,7 @@
     private SpotifyAppRemote mSpotifyAppRemote;
     private Connector.ConnectionListener mSpotifyRemoteConnectionListener;
     private Stack<Promise> mConnectPromises = new Stack<Promise>();
+    private ReadableMap mPlayerContext;
```

Comment:
I don't think this is being used anymore.  Was probably from your initial implementation?